models.py

```diff
@@ -0,0 +1,83 @@
+"""This module contains model related code"""
+
+from collections import Counter
+
+import lightgbm as lgb
+from sklearn.model_selection import StratifiedKFold
+import numpy as np
+
+SEED = 997
```

Comment:
Why are we setting seed as a set value?

Reply:
If you mean "set value" as "constant" then it's for reproducibility sake. If you mean something else please clarify :)